taittirIyam/sUtram/baudhAyanaH/brahma-karma-samuchchayaH/1_AhnikakANDam/14_devapUjAvidhiH.md

```diff
@@ -24,65 +24,67 @@ title = "१४ देवपूजाविधिः"
+व्यस्कभ्नाद्रोदसी विष्णुरेते दाधार पृथिवीमभितो मयूखैः ॥   
 
-विष्णवे नमः अक्षतान् स० ॥  
+विष्णवे नमः अक्षतान् स॰ ॥  
 
 > ॐ तद्विष्णोः परमं पद सर्दा पश्यन्ति सूरयः । दिवीव चक्षुराततम् ॥  
 
-> इमे पुष्पाः शुभा दिव्याः सर्वपुष्पैरलङ्कताः ।  
 पूता ब्रह्मपवित्रेण पूताः सूर्यस्य रश्मिभिः॥   
 
-विष्णवे नमः पुष्पाणि स० ॥  
+विष्णवे नमः पुष्पाणि स॰ ॥  
 
-ॐ तत्सवितुः० ॥  
+ॐ तत्सवितुः॰ ॥  
 
 > वनस्पतिरसो धूपो धूपेभ्यो धूप उत्तमः ।  
 आघ्रेयः सर्वदेवानां धूपोऽयं प्रतिगृह्यताम् ॥  
 
-विष्णवे नमः धूपं स० ॥  
+विष्णवे नमः धूपं स॰ ॥  
 
 > ॐ उद्दीप्यस्व जातवेदोऽपघ्नं निर्ऋतिं मम । पशूँश्च महामावह जीवनं च दिशो दिश ॥  
 
 > ज्योतिः शुक्रश्च तेजश्च देवानां सततं प्रियः ।  
 भास्वरः सर्वभूतानां दीपोऽयं प्रतिगृह्यताम् ॥  
 
-विष्णवे नमः दीपं स० ॥  
+विष्णवे नमः दीपं स॰ ॥  
 
-देवस्य पुरतश्चतुरश्रमण्डले महाहविः सर्वं स्वादुवस्तु-कन्दमूलफलानि च संस्थाप्याभिघार्य, गायत्र्या' प्रोक्ष्य धेनुमुद्रां प्रदर्श्य अर्योदकेनान्वितं पुष्पाञ्जलिं 'ॐ देवस्य त्वा सवितुः प्रसवेऽश्विनोर्बाहुभ्यं पूष्णो हस्ताभ्याम् ॥ विष्णवे नमः नैवेद्यार्थे इदं हविः क्षीरदधिफलमूलादिनैवेद्यं निवेदयामि स्वाहा' इति परिषिच्य समर्प्य, अङ्गुष्ठानामिकायोगेन नैवेद्यमुद्रां प्रदर्श्य, 'अमृतोपस्तरणमसि स्वाहा' इत्यर्घ्यं दत्वा, अङ्गुष्ठकनिष्ठानामिकाभिः 'ॐ प्राणाय स्वाहा', अङ्गुष्ठतर्जनीमध्यमाभिः 'ॐ अपानाय स्वाहा', मध्यमानामाङ्गुष्ठैः 'ॐ व्यानाय स्वाहा', अङ्गुष्ठतर्जनीमध्यमानाभिः ॐ उदानाय स्वाहा', सर्वाङ्गुलीभिः ॐ समानाय स्वाहा', इति पञ्च मुद्राः प्रदर्श्य, पटान्तरितो भुञ्जानं देवं मनसा ध्यायन् 'ॐ विष्णवे नमः' मध्ये पानीयं समर्पयामि । 'अङ्गुष्ठमात्रः पुरुषोङ्गुष्ठं च समाश्रितः । ईशः सर्वस्य जगतः प्रभुः प्रीणाति विश्वभुक्' समर्पितनैवेद्यं विसर्जयामि, 'अमृता पिधानमसि स्वाहा' इत्यर्घ्योदकं विसृज्य, नैवेद्यमुद्धृत्य, ॐ विष्णवे नमः आचमनीयं स० ॥ पुष्पैराराधयेत्- 'ॐ केशवाय नमः । नारायणाय । माधवाय० । गोविन्दाय० । विष्णवे० । मधुसूदनाय० । त्रिविक्रमाय० । वामनाय० । श्रीधराय० । हृषीकेशाय । पद्मनाभाय० । दामोदराय नमः' इति ॥ 
+देवस्य पुरतश्चतुरश्रमण्डले महाहविः सर्वं स्वादुवस्तु-कन्दमूलफलानि च संस्थाप्याभिघार्य, गायत्र्या प्रोक्ष्य धेनुमुद्रां प्रदर्श्य अर्योदकेनान्वितं पुष्पाञ्जलिं 'ॐ देवस्य त्वा सवितुः प्रसवेऽश्विनोर्बाहुभ्यं पूष्णो हस्ताभ्याम् ॥ विष्णवे नमः नैवेद्यार्थे इदं हविः क्षीरदधिफलमूलादिनैवेद्यं निवेदयामि स्वाहा' इति परिषिच्य समर्प्य, अङ्गुष्ठानामिकायोगेन नैवेद्यमुद्रां प्रदर्श्य, 'अमृतोपस्तरणमसि स्वाहा' इत्यर्घ्यं दत्वा, अङ्गुष्ठकनिष्ठानामिकाभिः 'ॐ प्राणाय स्वाहा', अङ्गुष्ठतर्जनीमध्यमाभिः 'ॐ अपानाय स्वाहा', मध्यमानामाङ्गुष्ठैः 'ॐ व्यानाय स्वाहा', अङ्गुष्ठतर्जनीमध्यमानाभिः ॐ उदानाय स्वाहा', सर्वाङ्गुलीभिः ॐ समानाय स्वाहा', इति पञ्च मुद्राः प्रदर्श्य, पटान्तरितो भुञ्जानं देवं मनसा ध्यायन् 'ॐ विष्णवे नमः' मध्ये पानीयं समर्पयामि । 'अङ्गुष्ठमात्रः पुरुषोङ्गुष्ठं च समाश्रितः । ईशः सर्वस्य जगतः प्रभुः प्रीणाति विश्वभुक्' समर्पितनैवेद्यं विसर्जयामि, 'अमृता पिधानमसि स्वाहा' इत्यर्घ्योदकं विसृज्य, नैवेद्यमुद्धृत्य, ॐ विष्णवे नमः आचमनीयं स॰ ॥ पुष्पैराराधयेत्- 'ॐ केशवाय नमः । नारायणाय । माधवाय॰ । गोविन्दाय॰ । विष्णवे॰ । मधुसूदनाय॰ । त्रिविक्रमाय॰ । वामनाय॰ । श्रीधराय॰ । हृषीकेशाय । पद्मनाभाय॰ । दामोदराय नमः' इति ॥ 
```

Comment:
अत्र दीर्घोद्धरणानि यथोपदेशं विन्यस्यतु

Reply:
विहितानि, तथैव वा इति ज्ञातुं इच्छामि

Reply:
naivam. īkṣatām, īmāny ēva dīrghōddharaṇē - 

- 'ॐ देवस्य त्वा सवितुः प्रसवेऽश्विनोर्बाहुभ्यं पूष्णो हस्ताभ्याम् ॥ विष्णवे नमः नैवेद्यार्थे इदं हविः क्षीरदधिफलमूलादिनैवेद्यं निवेदयामि स्वाहा'
- 'ॐ केशवाय नमः । नारायणाय । माधवाय॰ । गोविन्दाय॰ । विष्णवे॰ । मधुसूदनाय॰ । त्रिविक्रमाय॰ । वामनाय॰ । श्रीधराय॰ । हृषीकेशाय । पद्मनाभाय॰ । दामोदराय नमः'

aśiṣṭaṁ akhilam alpōddharaṇaṁ vā, anuddharaṇaṁ vā. 

ēvaṁ vivicya tadanusārēṇa vinyasyatu.

Reply:
अत्र यत् bullet स्थापितम् खलु कथं तस्य स्थापनं?

Reply:
मत्सन्देशे 'ॐ देवस्य त्वा सवितुः' इत्यस्माद् पूर्वम् आदौ दृश्यमानं चिह्नम्? तत् त्व् अनपेक्षितम् अत्र (पाठशोधनदृष्ट्या)। केवलं मया सन्देशे स्पष्टतायै प्रयुक्तम्। तच्च पङ्क्तेर् आदौ `-`-चिह्नस्योट्टङ्कनेन लभ्यते (यथात्र प्रतिपादितम् - https://sanskrit.github.io/groups/dyuganga/projects/text/proofreading/editing/markdown/#headings-and-sections )।

Reply:
अत्रापि उक्तरीत्या कर्तुं प्रयत्नः कृतः